alerts/cluster-monitoring-operator/PrometheusRemoteStorageFailures.md

```diff
@@ -0,0 +1,43 @@
+# PrometheusRemoteStorageFailures
+
+## Meaning
+
+The `PrometheusRemoteStorageFailures` alert triggers when there has been
+a constant increase in failures sending samples to the remote storage for more
+than 15 minutes, both in plaform Prometheus or in user-defined Prometheus.
+
+## Impact
+
+Prometheus samples in storage might be missing or inaccurate.
+
+## Diagnosis
+
+1. Determine whether the alert has triggered for the instance of Prometheus used
+for default cluster monitoring or for the instance that monitors user-defined
+projects by viewing the alert message's `namespace` label: the namespace for
+default cluster monitoring is `openshift-monitoring`; the namespace for user
+workload monitoring is `openshift-user-workload-monitoring`.
+
+2. Review the logs for the affected Prometheus instance:
+
+    ```console
+    $ NAMESPACE='<value of namespace label from alert>'
+
+    $ oc -n $NAMESPACE logs -l 'app=prometheus'
+    level=error ... msg="Failed to send batch, retrying" ...
```

Comment:
I tested this on a cluster and `app=prometheus` gives me `No resources found in openshift-monitoring namespace.` 
did you mean `oc logs -l 'app.kubernetes.io/component=prometheus'`